src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -43,6 +43,8 @@ public class DocumentStore : DocumentStoreBase
 
         private string _identifier;
 
+        public AsyncMultiDatabaseHiLoIdGenerator AsyncMultiDatabaseHiLoIdGenerator => _asyncMultiDbHiLo;
```

Comment:
I don't like this name. Maybe just `HiloIdGenerator` ?

Reply:
actually `IHiLoIdGenerator` and expose only methods that interest you. This class contains various methods that you probably would like to avoid exposing to public API

Reply:
Also (again) you are not adding that to the `IDocumentStore` interface

Reply:
done